### ISSUE TITLE
0.12.0-RC5: image doesn't refresh

### ISSUE THREAD

**Issue by ChristianMayer** [CometVisu/CometVisu#1186]
Running RC5 the yr.no meteogram (SVG version) on the wall mounted touch panels doesn't update, i.e. it's always showing the time of the browser start.

Trying to replicate the same behavior with my usual machine it does work with RC5 as well as the current develop version.

Touch panel is a RPi with Chrome, my work machine is x86 with Chrome.

Bug is critical as using the image widget with exactly this type of image and refresh is a very common use case.

Config snippet:
```
    <image src="https://www.yr.no/en/content/2-2899676/meteogram.svg" width="100%" widthfit="true" refresh="100" cachecontrol="force">
      <layout colspan="12"/>
    </image>
    ```

**Comment by ChristianMayer** [comment]
It's not related to #1179 as the issue happened even earlier, I guess all 0.12.0 are affected.
Actually by creating #1179 I hoped that this fixes this issue as well... At least it contains a fix for the broken state that 0.11.2 already had. But I can't confirm as I couldn't trigger it on purpose.

About the invalid HTML I'll investigate. Whether it's a solution we'll most likely only know after the release as I can't reproduce the missing reload on my development machine.

**Comment by ChristianMayer** [comment]
Please note that the `iframe` gets already removed at line https://github.com/CometVisu/CometVisu/blob/0e0567c6a5c2497f0223293fc9b9205a41262a85/source/class/cv/ui/common/Refresh.js#L264

The leftovers of line 273 are a bug.

**Comment by peuter** [comment]
Well then the removal does not work, using your example with a shorter refresh timer (5) shows that every refresh a new iframe is added and none is removed.

![image](https://user-images.githubusercontent.com/5507035/152553438-6ddc2635-6436-4bd1-93ae-4f0da08e87ed.png)

**Comment by peuter** [comment]
The loadCallback is never called, probably because the listeners are added after the iframe has been added to the body, so they might never fire.

**Comment by ChristianMayer** [comment]
I can not reproduce the issue about leaking `iframe`s. 
I tried with and without cache. I also tried with all three values for `cachecontrol`

**Comment by peuter** [comment]
With the fixes from #1192 the iframe removal is working.
